<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Editmaincompany</class>
 <widget class="QDialog" name="Editmaincompany">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>619</width>
    <height>279</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>600</width>
     <height>180</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>74</y>
      <width>571</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="Lblcnpj">
       <property name="text">
        <string>CNPJ</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="LnEdtcnpj">
       <property name="inputMask">
        <string>99.999.999/9999-99</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Lblie">
       <property name="text">
        <string>Inscrição Estadual</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="LnEdtie"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>9</y>
      <width>571</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="Lblname">
       <property name="minimumSize">
        <size>
         <width>71</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Razão Social</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="LnEdtname"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>44</y>
      <width>571</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="Lblfantasyname">
       <property name="text">
        <string>Nome Fantasia</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="LnEdtfantasyname"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>114</y>
      <width>571</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="Lblphone">
       <property name="text">
        <string>Telefone</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="LnEdtphone">
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Lblemail">
       <property name="text">
        <string>email</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="LnEdtemail"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>144</y>
      <width>571</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="Lblcity">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>42</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Cidade</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="SComboBox" name="CmbBxcity">
       <property name="editable">
        <bool>true</bool>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QComboBox::AdjustToContents</enum>
       </property>
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Lblstate">
       <property name="maximumSize">
        <size>
         <width>42</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Estado</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="SComboBox" name="CmbBxstate">
       <property name="editable">
        <bool>true</bool>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QComboBox::AdjustToContents</enum>
       </property>
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QFrame" name="topmenu">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>1</y>
     <width>801</width>
     <height>42</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>42</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>42</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(50, 0, 133);</string>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="labelStatus">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>0</y>
      <width>541</width>
      <height>39</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>50</red>
          <green>0</green>
          <blue>133</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>50</red>
          <green>0</green>
          <blue>133</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>50</red>
          <green>0</green>
          <blue>133</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>50</red>
          <green>0</green>
          <blue>133</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>50</red>
          <green>0</green>
          <blue>133</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>50</red>
          <green>0</green>
          <blue>133</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>50</red>
          <green>0</green>
          <blue>133</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>50</red>
          <green>0</green>
          <blue>133</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>50</red>
          <green>0</green>
          <blue>133</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string/>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string>Cadastro da Empresa</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::NoTextInteraction</set>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="PshBtnCancel">
   <property name="geometry">
    <rect>
     <x>438</x>
     <y>240</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Cancelar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PshBtnSave">
   <property name="geometry">
    <rect>
     <x>525</x>
     <y>240</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Salvar</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">scombobox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>LnEdtname</tabstop>
  <tabstop>LnEdtfantasyname</tabstop>
  <tabstop>LnEdtcnpj</tabstop>
  <tabstop>LnEdtie</tabstop>
  <tabstop>LnEdtphone</tabstop>
  <tabstop>LnEdtemail</tabstop>
  <tabstop>CmbBxcity</tabstop>
  <tabstop>CmbBxstate</tabstop>
  <tabstop>PshBtnCancel</tabstop>
  <tabstop>PshBtnSave</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
